src/components/pages/compare/comparison-cell.tsx

```
@@ -0,0 +1,33 @@
+    return <Check className="size-4 text-emerald-500" aria-label="Supported" />;
+  }
+
+  if (support === "no") {
+    return <X className="size-4 text-rose-500" aria-label="Not supported" />;
+  }
+
+  return (
+    <TooltipProvider>
+      <Tooltip>
+        <TooltipTrigger asChild>
+          <span className="inline-flex cursor-help items-center gap-1 text-amber-400" aria-label="Partially supported">
```

Biome (2.4.12)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)